src/less-style/question-props.less

```diff
@@ -157,3 +157,17 @@
   padding: 0 2em;
   margin-bottom: 2em;
 }
+
+.fake-textarea {
```

Comment:
Yeah. I'm going to fix that properly later. Probably going to do it in HQ so that I can be sure it inherits all the correct styles and still looks like textarea/input